networking/nw-configuring-ingress-controller-endpoint-publishing-strategy.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="nw-configuring-ingress-controller-endpoint-publishing-strategy"]
+= Configuring the Ingress Controller endpoint publishing strategy
+include::_attributes/common-attributes.adoc[]
+:context: nw-configuring-ingress-controller-endpoint-publishing-strategy
+
+toc::[]
+
+The `endpointPublishingStrategy` is used to publish the Ingress Controller endpoints to other networks, enable load balancer integrations, and provide access to other systems.
+
+[IMPORTANT]
+====
+On {rh-openstack-first}, the `LoadBalancerService` endpoint publishing strategy is only supported if a cloud provider is configured to create health monitors. For {rh-openstack} 16.2, this strategy is only possible if you use the Amphora Octavia provider.
```

Comment:
"Only" needs to be as close as possible to the term it modifies.

s/only supported if a cloud provider / supported only if a cloud provider
s/ only possible if you use / possible only if you use

Reply:
:heavy_check_mark: